### DISCUSSION TITLE
Participating in Hacktoberfest 2021

### DISCUSSION THREAD

**Discussion by RajMhatre20** [sButtons/sbuttons#1488] [Ideas]
How about participating in hacktoberfest 2021? The repository will get more attention and more new enthusiast will be able to find and contribute to this repo.

**Comment by shahednasser** [comment]
Hi @RajMhatre20 the repository is already participating in hacktoberfest as it has the `hacktoberfest` label. However, there aren't that many issues to contribute to. People would have to submit issues or button ideas to work on.

**Reply by RajMhatre20** [reply]
Yup sorry, I found it later, I couldn't find a way to delete the discussion.

**Reply by shahednasser** [reply]
no worries, thank you for asking as I'm sure others will have the same question!
